test/image/mocks/icicle_leaf-opacity-level.json

```diff
@@ -0,0 +1,84 @@
+{
+  "data": [
+    {
+      "type": "icicle",
+      "labels": ["Eve", "Cain", "Seth", "Enos", "Noam", "Abel", "Awan", "Enoch", "Azura"],
+      "parents": ["", "Eve", "Eve", "Seth", "Seth", "Eve", "Eve", "Awan", "Eve" ],
+      "domain": {"x": [0, 0.20], "y": [0, 0.9]},
+      "root": {"color": "limegreen"},
+      "level": "",
+      "leaf": {
+        "opacity": 0.1
```

Comment:
Perhaps it would be better to use the highest leaf.opacity for this one and then reduce it?

Reply:
Hmm that could work. I'd be in favour of reversing the order to make it visually apparent what is going on.

Reply:
I just reversed it and it looks like this locally:

![image](https://user-images.githubusercontent.com/10369095/112545166-43f60080-8d8e-11eb-882a-613f98dd4aae.png)


But looking at this mock test again, I think it would be even better for demonstrative purposes to create a grid of examples. I think it would be most helpful to show all the combinations of `(level, opacity)`:

eg. 
```
[ ] [ ] [ ] [ ]
[ ] [ ] [ ] [ ]
[ ] [ ] [ ] [ ]
[ ] [ ] [ ] [ ]
```

@archmoj Do you think this makes sense, or is it overkill?

Reply:
A 4x4 grid might be overkill.
You could keep it as is or add a 2x2 or 3x3 gird, if you want.

Reply:
Okay sounds good.